src/app/modules/forms/forms.js

```diff
@@ -862,17 +862,27 @@ formsModule.directive('hodSortcode', ['FormsService', function (FormsService) {
         }
 
         scope.isValid = function () {
-          var pt1 = Number(scope.data.part1)
-          var pt2 = Number(scope.data.part2)
-          var pt3 = Number(scope.data.part3)
+          var sortCodeArray = [
+            Number(scope.data.part1),
+            Number(scope.data.part2),
+            Number(scope.data.part3)
+          ]
+
+          var allZeroSortCode = sortCodeArray.every(function(value) {
+            return value === 0
+          })
+
+          var methodToName = sortCodeArray.every(function(value) {
```

Comment:
Why is this function called `methodToName`? It might be better to rename this to something more descriptive.